.github/workflows/ci.yml

```diff
@@ -75,7 +75,7 @@ jobs:
       - name: Build package for upload to PyPI
         run: python -m build .
       - name: Upload the distribution to PyPI
-        uses: pypa/gh-action-pypi-publish@v1.4.2
+        uses: pypa/gh-action-pypi-publish@v1.13.0
```

Comment:
Last time I tried anything newer than 1.4.2 was not allowed in @mozilla org:/

It errors out in this startup failure: https://github.com/mozilla/django-product-details/actions/runs/18003443633